C/ConnectFourSolver/bundled/patches/generator.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/generator.cpp b/generator.cpp
+index 7afeb8e..327c3ad 100644
+--- a/generator.cpp
++++ b/generator.cpp
+@@ -81,7 +81,7 @@ void generate_opening_book() {
+ int main(int argc, char** argv) {
+   if(argc > 1) {
+     int depth = atoi(argv[1]);
+-    char pos_str[depth + 1] = {0};
++    char *pos_str = new char[depth + 1](); // zero initialized
```

Comment:
I guess this fix should be submitted upstream...

Reply:
Summitted at https://github.com/PascalPons/connect4/pull/20   
I'll merge this patch in my fork this weekend if it doesn't get merged by then.

Reply:
Updated my fork. Applying this change should work now. https://github.com/fingolfin/Yggdrasil/pull/1/files